### DISCUSSION TITLE
Predicted probabilities

### DISCUSSION THREAD

**Discussion by ZoGo94** [biomodhub/biomod2#419] [Q&A]
Different modelling algorithms produce different probability of occurrence (as might be expected) but what I want to know is whether these are comparable? i.e. is a probability of occurrence of 50% for one pixel for one algorithm (say ANN) predicting lower likelihood of occurrence than 80% (say for random forest)? And the probability output is on a scale of 0-1000, right, so 500 versus 800?  I want to know if I can create a map of standard deviation of probability across multiple algorithms that represents uncertainty between algorithms.

**Comment by HeleneBlt** [comment]
Hello ! 

Thanks for your comment 🙏 
It seems you are looking to `EMcv`, one of the results of the `BIOMOD_EnsembleModeling` function. It calculates the coefficient of variation (sd / mean) of probabilities over the selected models
This model is not scaled. It will be evaluated like all other ensemble models although its interpretation will be obviously different. CV is a measure of uncertainty rather a measure of probability of occurrence. If the CV gets a high evaluation score, it means that the uncertainty is high where the species is observed (which might not be a good feature of the model). The lower is the score, the better are the models. CV is a nice complement to the mean probability. 

Hope it helps. 
Hélène

**Reply by ZoGo94** [reply]
Thank you for your response.  Am I right in thinking that if this calculates the coefficient of variation of probabilities across different models (which could be generated by different algorithms) then the probabilities are indeed comparable between algorithms?  What if I generated a pool of models using 8 different algorithms and generated 8 ensembles by algorithm and wanted to look at the uncertainty between these 8 generated ensemble models, for example?

**Reply by HeleneBlt** [reply]
Yes, the probabilities are comparable between algorithms. However, you have to pay attention to how you merge your models for your ensemble models. If you do `em.by = 'all'`, you're gonna merge the models from different algorithms but also the different sets of pseudo-absences and cross-validation, so the uncertainty will not come only from the algorithms. You can have a look at the documentation of [BIOMOD_EnsembleModeling](https://biomodhub.github.io/biomod2/reference/BIOMOD_EnsembleModeling.html).

Hélène
